chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -225,10 +225,10 @@ fi
 
 # for making HA on shared NFS backend storage work
 add_group glance 200
-add_user glance 200 glance
 add_group qemu 201
-add_user qemu 201 kvm
 add_group kvm 202
+add_user glance 200 glance
+add_user qemu 201 kvm
 
```

Comment:
That's what I though at first too, but line 231 depends on line 229 and simply moving the `add_user ... kvm` line up would break the order in the uid/gid values. So for aesthetic reasons I opted to order by uid/gid values and grouped by `add_{user,group}` commands :smile:.